gettingstarted.html

```diff
@@ -0,0 +1,114 @@
+    <img class="col col-2-3" src="/assets/gettingstartedframe1.png">
+    <p class="col col-1-3 pcenter">
+      On home, click the XO Icon, and choose My Settings.<br>
+    </p>
+  </div>
+
+  <div class="grid">
+    <p class="col col-1-3">
+      In the Settings Window, select Frame Icon.
+    </p>
+    <img class="col col-2-3" src="/assets/gettingstartedframe2.png" padding=0> 
+  </div>
+
+  <div class="grid">
+    <img class="col col-2-3" src="/assets/gettingstartedframe3.png">
+    <p class="col col-1-3 pcenter">
+      In the Activation Delay, you can choose the delay it takes to show the frame from the edges as well as corners.
+    </p>
+  </div>
+
+<h1>Using Journal</h1>
+  <div class="grid">
+    <img class="col col-2-3" src="/assets/gettingstartedframejournal.png">
+    <p class="col col-1-3">
+      From the frame, Click on the Journal button.<br>
+    </p>
+  </div>
+  <div class="grid">
+    <p class="col col-1-3">
+      This will open up the Journal.<br>
+      You can think of this as call history, but a much more sophisticated, usable and powerful. It logs all the times you open an activity and what you do inside it. You can click on any instance and add/view/edit the description.
+    </p>
+    <img class="col col-2-3" src="/assets/gettingstartedjournal.png">
+  </div>
+    
+
+<h1>Connect to a Wi-Fi</h1>
+    <div class="grid">
+      <img class="col col-2-3" src="/assets/gettingstarted-1.png">
+      <p class="col col-1-3">
+        Take the mouse to one of the corners, in order to reveal the frame.<br>
+      </p>
+    </div>
+
+    <div class="grid">
+      <p class="col col-1-3">
+        Click on the Neighborhood button from the frame.<br>
+      </p>
+      <img class="col col-2-3" src="/assets/gettingstarted-2.png">
+    </div>
+
+    <div class="grid">
+      <img class="col col-2-3" src="/assets/gettingstarted-3.png" padding=0> 
+      <p class="col col-1-3">
+        Search for the Wi-Fi SSID from amongst all the circles shown.<br>
+        Click on the Wi-Fi and select Connect.
+      </p>
+    </div>
+
+    <div class="grid">
+      <p class="col col-1-3">
+        If it is the first time use, it will ask for the Wi-Fi password.<br>
+        Entering that will connect you to the Wi-Fi successfully.
+      </p>
+      <img class="col col-2-3" src="/assets/gettingstarted-4.png">
+    </div>
+
+<h1>Collaboration</h1>
+  <div class="grid">
+    <img class="col col-2-3" src="/assets/gettingstartedcollaboration2.png">
+    <p class="col col-1-3">
+      After connecting to the Wi-Fi, go to the Neighborhood view. Here you will find a huge number of XO icons on the screen. You can add the people you want to collaborate with, as friends from this screen.<br>
+    </p>
+  </div>
+  <div class="grid">
+    <p class="col col-1-3">
+      Now to share with the help of collaboration, go to the Journal and click the activity instance you want to share. In Send To, you will find the people you can collaborate with.
```

Comment:
No, that using the send file function in the journal sends the file to the user - it is like emailing them a copy of the file.

Please try playing with collaboration before writing a tutorial on it.  I can help you if you need to set it up.  But you can defiantly do it with sugar build, if the jabber server option is set to "" (empty text box) - which means collaborate over the local network.